acceptance/lib/pxp-agent/config_helper.rb

```diff
@@ -66,11 +66,11 @@ def pxp_config_hocon(broker, agent, ssl_config = {})
   })
 end
 
-def pxp_config_hocon_using_puppet_certs(broker, agent, num_brokers=1)
-  to_hocon(pxp_config_hash_using_puppet_certs(broker, agent, num_brokers))
+def pxp_config_hocon_using_puppet_certs(broker, agent, num_brokers=1, master_proxy: "", broker_proxy: "")
```

Comment:
This might be a good time to make `num_brokers` optional too, i.e. `num_brokers: 1`.